examples/timeseries_convolution_filter.py

```diff
@@ -0,0 +1,56 @@
+"""
+======================================================
+Smoothing of TimeSeries Data Using Convolution Filters
+======================================================
+
+Sometimes, when working with scientific data, we have noisy data from which
+we need to extract low-frequency components from. This example illustrates
+creating a TimeSeries data, adding noise to it and then smoothing it using
+convolution filter kernels from `~astropy.convolution` and 
+`~astropy.convolution.convolve` function.
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+import datetime
+
+import numpy as np
+import matplotlib.pyplot as plt
+from pandas import DataFrame
+
+from astropy.convolution import convolve, Box1DKernel
+
+from sunpy.timeseries import TimeSeries
+
+###############################################################################
+# We will first create a sine wave with time range spanning over 1 day and then
+# convert it into a `TimeSeries` object by passing as pandas DataFrame.
+
+base = datetime.datetime.today()
+times = [base - datetime.timedelta(minutes=x) for x in range(0, 24 * 60)]
+intensity = np.sin(np.arange(0, 12 * np.pi, step=(12 * np.pi) / (24 * 60)))
+original_df = DataFrame(intensity, index=times, columns=['intensity'])
+# Convert into TimeSeries
+original_ts = TimeSeries(original_df)
+original_ts.peek(title="Original TimeSeries")
+
```

Comment:
Have you looked at any of the sample data to create the signal instead?

Reply:
A noisy sample data? Perhaps I can use the one of those which already have a smoothed data along with original data and then compare them both (the one we have and the one calculated) ? like `sunpy.data.sample.NOAAINDICES_TIMESERIES` used in the peak finding one.

Reply:
Something like that would be a good idea.